### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

**Comment by bearnard** [comment]
👍

File: tests/test_data_models.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+def fetch_faqs(dbsession: Session) -> list[FAQModel]:
+    query = select(FAQModel).order_by(FAQModel.faq_id)
+    return list(dbsession.execute(query).scalars())
+
+
+def test_faq(dbsession):
+    """
+    We can create an FAQ entry and load it from the db.
```
Comment (bearnard) [inline tests/test_data_models.py]: `a`?

Comment (jerith) [reply tests/test_data_models.py]: "[...] an eff ay kew entry [...]" 😝

(I'll fix that when I update the test.)

**Comment by bearnard** [comment]
👍

**Comment by bearnard** [comment]
👍

**Pull request merged**